### PR TITLE
Add back penguin sklearn to pre-commit checks

### PR DESCRIPTION
**Pull request by casassg**
Related to #61

### CODE REVIEW

**Comment by TheMichaelHu** [comment]
Oh that's funny, yapf caused lint errors. https://github.com/tensorflow/tfx-addons/pull/62/files#diff-64892c783cb937313fc35f3e3c429f867f78aad2e70afde0a88390bd3d4a1822R192

**Comment by casassg** [comment]
Yeah, I can fix those and merge :P will do tomorrow though

**Comment by casassg** [comment]
@TheMichaelHu Can you review and merge if possible?

File: .github/ci/.pylintrc [hunk 1]
```diff
@@ -41,7 +41,7 @@ enable=indexing-exception,old-raise-syntax
...
 # --disable=W"
 # TODO(b/154672575): Enable wrong-import-order.
-disable=design,similarities,no-self-use,attribute-defined-outside-init,locally-disabled,star-args,pointless-except,bad-option-value,global-statement,fixme,suppressed-message,useless-suppression,locally-enabled,no-member,no-name-in-module,import-error,unsubscriptable-object,unbalanced-tuple-unpacking,undefined-variable,not-context-manager,useless-object-inheritance,unnecessary-pass,no-else-return,no-else-raise,no-else-continue,wrong-import-order,no-value-for-parameter,misplaced-comparison-constant,no-init,abstract-method,unpacking-non-sequence
+disable=design,similarities,no-self-use,attribute-defined-outside-init,locally-disabled,star-args,pointless-except,bad-option-value,global-statement,fixme,suppressed-message,useless-suppression,locally-enabled,no-member,no-name-in-module,import-error,unsubscriptable-object,unbalanced-tuple-unpacking,undefined-variable,not-context-manager,useless-object-inheritance,unnecessary-pass,no-else-return,no-else-raise,no-else-continue,wrong-import-order,no-value-for-parameter,misplaced-comparison-constant,no-init,abstract-method,unpacking-non-sequence,super-with-arguments
```
Comment (TheMichaelHu) [inline .github/ci/.pylintrc:44]: If you're adding this just because of the sklearn example, I can fix the super issues and remove this in a separate PR.

**Review by TheMichaelHu — APPROVED** [review]

File: .github/ci/.pylintrc (hunk 1 above)
Comment (casassg) [reply .github/ci/.pylintrc:44]: I did, I also added it as it's a bit to restrictive imo. I use both super styles most of the times. That said, if we want to keep in sync w TFX repo, we should remove it

**Comment by TheMichaelHu** [comment]
Looks like I don't have permission to merge this PR for you.

**Review by rcrowe-google — APPROVED** [review]
Thanks!

**Pull request merged**